### PR TITLE
Develop 

### PR DESCRIPTION
**Pull request by Dimoshua**
Request description:
- [DEMO LINK](https://Dimoshua.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Dimoshua.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

**Comment by Dimoshua** [comment]
Request description:
- [DEMO LINK](https://Dimoshua.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Dimoshua.github.io/layout_html-form/report/html_report/)

File: src/index.html [hunk 1]
```diff
@@ -4,14 +4,236 @@
...
+      oninput="result.value=myrange.value"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label for="surname" class="block">
+          Surname:
+          <input
+            type="text"
+            id="surname"
+            name="Personal_information"
```
Comment (agniya-i) [inline src/index.html:29]:
```suggestion
            name="surname"
```

File: src/index.html [hunk 2]
```diff
@@ -4,14 +4,236 @@
...
+            required
+          >
+        </label>
+
+        <label for="name" class="block">
+          Name:
+          <input
+            type="text"
+            id="name"
+            name="Personal_information"
```
Comment (agniya-i) [inline src/index.html:40]:
```suggestion
            name="name"
```

File: src/index.html [hunk 3]
```diff
@@ -4,14 +4,236 @@
...
+            required
+          >
+        </label>
+
+        <label for="age" class="block">
+          How old are You?
+          <input
+            type="number"
+            id="age"
+            name="Personal_information"
```
Comment (agniya-i) [inline src/index.html:51]:
```suggestion
            name="age"
```

File: src/index.html [hunk 4]
```diff
@@ -4,14 +4,236 @@
...
+        </label>
+
+        <label for="dateOfBirth" class="block">
+          Full date of birth:
+          <input
+            type="date"
+            id="dateOfBirth"
+            min="01.01.1922"
+            max="01.01.2022"
+            name="Personal_information"
```
Comment (agniya-i) [inline src/index.html:66]:
```suggestion
            name="dateOfBirth"
```

File: src/index.html [hunk 5]
```diff
@@ -4,14 +4,236 @@
...
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <label for ="myemail" class="block">
+          E-mail:
+          <input
+            type="email"
+            id="myemail"
```
Comment (agniya-i) [inline src/index.html:88]:
```suggestion
            id="email"
```

File: src/index.html [hunk 6]
```diff
@@ -4,14 +4,236 @@
...
+            name="Registration"
```
Comment (agniya-i) [inline src/index.html:89]:
```suggestion
            name="email"
```

File: src/index.html [hunk 7]
```diff
@@ -4,14 +4,236 @@
...
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+        <div class="block">Do you love cats?
+          <label for="yescats">
+            <input type="radio"
```
Comment (agniya-i) [inline src/index.html:113]:
```suggestion
            <input 
              type="radio"
```
start first property also on new line

File: src/index.html [hunk 8]
```diff
@@ -4,14 +4,236 @@
...
+              id="yescats"
+              name="Interesting_fact"
```
Comment (agniya-i) [inline src/index.html:115]:
```suggestion
              name="yes"
```

File: src/index.html [hunk 9]
```diff
@@ -4,14 +4,236 @@
...
+        </label>
+
+        <label for="myrange" class="block">
+          How do you rate our work?
+          <input
+            type="range"
+            id="myrange"
+            value="0"
+            step="5"
+           required
```
Comment (agniya-i) [inline src/index.html:210]:
```suggestion
             required
```

File: src/index.html [hunk 10]
```diff
@@ -4,14 +4,236 @@
...
+            step="2"
+            name="Interesting_fact"
+            required
+          >
+        </label>
+
+        <label class="block">
+          What are your favorite brand of cars?
+          <select required multiple>
+            <option value="BMW">BMW</option>
```
Comment (agniya-i) [inline src/index.html:157]: leave only 3 options, as on the task screenshot  🙂

**Review by agniya-i — CHANGES_REQUESTED** [review]
pay attention to `name` property, use more meaningful names

**Pull request closed without merging**